grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -974,12 +975,19 @@ public GrpcService build() {
             registryBuilder.addService(grpcHealthCheckService.bindService(), null, ImmutableList.of());
         }
 
-        final GrpcExceptionHandlerFunction grpcExceptionHandler;
+        if (exceptionHandler == null) {
+            exceptionHandler = GrpcExceptionHandlerFunction.ofDefault();
+        } else {
+            exceptionHandler = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault());
```

Comment:
ditto

Reply:
Is this ditto to https://github.com/line/armeria/pull/5571#discussion_r1591771909?

Reply:
Yes we can probably do:
```java
GrpcExceptionHandlerFunction grpcExceptionHandler;
if (exceptionMappingsBuilder != null) {
    grpcExceptionHandler = exceptionMappingsBuilder.build().orElse(GrpcExceptionHandlerFunction.of());
} else if (exceptionHandler != null) {
    grpcExceptionHandler = exceptionHandler.orElse(GrpcExceptionHandlerFunction.of());;
} else {
    grpcExceptionHandler = GrpcExceptionHandlerFunction.of();
}
grpcExceptionHandler = new UnwrappingGrpcExceptionHandleFunction(grpcExceptionHandler);
registryBuilder.setDefaultExceptionHandler(grpcExceptionHandler);

if (interceptors != null) {
    final HandlerRegistry.Builder newRegistryBuilder = new HandlerRegistry.Builder();
    final ImmutableList<ServerInterceptor> interceptors = this.interceptors.build();
    for (Entry entry : registryBuilder.entries()) {
        final MethodDescriptor<?, ?> methodDescriptor = entry.method();
        final ServerServiceDefinition intercepted =
                ServerInterceptors.intercept(entry.service(), interceptors);
        newRegistryBuilder.addService(entry.path(), intercepted, methodDescriptor, entry.type(),
                                      entry.additionalDecorators());
    }
    newRegistryBuilder.setDefaultExceptionHandler(grpcExceptionHandler);
    handlerRegistry = newRegistryBuilder.build();
} else {
    handlerRegistry = registryBuilder.build();
}
```

Reply:
Applied feedback, Thank.
Code gets cleaner.